src/Tizen.NUI.Components/Attributes/ViewAttributes.cs

```diff
@@ -363,6 +378,29 @@ private void OnThemeChangedEvent(object sender, StyleManager.ThemeChangeEventArg
             }
         }
 
+        public void Dispose()
+        {
+            StyleManager.Instance.ThemeChangedEvent -= OnThemeChangedEvent;
```

Comment:
Your current code requests a user to call Dispose() explicitly. But I think there's better way.
How about this?
```C#
// Register OnThemeChanged callback to StyleManager
public virtual void ApplyToView(View view, ControlStates state = ControlStates.Normal)
{
  ...
  if (!appliedViews.ContainsKey(view))
  {
    if (!appliedViews.Count)
    {
      StyleManager.Instance.ThemeChangedEvent += OnThemeChangedEvent;
    }

    appliedViews.Add(view, 0);
    RegisterStateChangeCallback(view);
  }
}
```
```C#
// Unregister OnThemeChanged callback from StyleManager
public void DisApplyToView(View view)
{
  if (appliedViews.ContainsKey(view))
  {
    appliedViews.Remove(view);

    if (!appliedViews.Count)
    {
      StyleManager.Instance.ThemeChangedEvent -= OnThemeChangedEvent;
    }
  }
}
```

Reply:
By the way, why the attribute wants to receive ThemeChanged event from the StyleManager in the first place?
From the code, when theme has changed, the attribute just apply its properties again to the view. Can you explain why you need it?

Reply:
Sorry to bother you, but can you answer [this](https://github.com/Samsung/TizenFX/pull/1085#issuecomment-543152739) also?

Reply:
"Register OnThemeChanged in Constructor -> ApplyToView
Unregister OnThemeChanged in Dispose -> DisApplyToView"

The Attributes need to register the OnThemeChanged only once, but ApplyToView() maybe called many times, so I think it is better to register it in the constructor.

The Style and Attributes will let the View perceive the change of style in theme, because we don't want to let BaseComponents.View depend the Tizen.NUI.Components, so that we must register the OnThemeChanged and apply different style.

The current code of OnThemeChanged() has problem, I'm modifying it.

Reply:
I understand your point, but from my code above,
it checks `if (!appliedViews.Count)` before register OnThemeChanged, after then, it increments `Count` by adding view to `appliedViews`.
So I believe that it won't happen that the OnThemeChanged  is registered multiple times.

And I hope that the Attributes class is not IDisposable, because users are not likely to call Dispose() method.